songr/src/main/java/com/lab11java401d3/songr/AlbumController.java

```diff
@@ -0,0 +1,39 @@
+package com.lab11java401d3.songr;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/albums")
+public class AlbumController {
+
+    @Autowired
+    AlbumRepository repo;
+
+    @GetMapping("/")
+    public Iterable<Album> getAlbums() {
+        Album album = new Album();
+        album.title = "Test Title";
+        album.artist = "Test Artist";
+        album.songCount = 11;
+        album.length = 120;
+        album.imageURL = "test/image/url";
+
+        this.repo.save(album);
+
+        Iterable<Album> albums = this.repo.findAll();
+
+        return albums;

```

Comment:
This returns the literal albums in the db. You'll need to return a template for the method to render all the albums on the page.